allensdk/brain_observatory/filtered_running_speed/filtered_running_speed.py

```diff
@@ -0,0 +1,379 @@
+import numpy as np
+import pandas as pd
+from allensdk.brain_observatory.sync_dataset import Dataset as SyncDataset
+from allensdk.brain_observatory import sync_utilities
+import argschema
+import json
+import h5py
+
+from allensdk.brain_observatory.filtered_running_speed._schemas import (
+    InputParameters,
+    OutputParameters
+)
+
+from allensdk.brain_observatory.behavior.data_objects.\
+    running_speed.running_processing import (
+        get_running_df
+    )
+
+
+class FilteredRunningSpeed(argschema.ArgSchemaParser):
+    default_schema = InputParameters
+    default_output_schema = OutputParameters
+
+    START_FRAME = 0
+
+    def _extract_dx_info(
+        self,
+        frame_times: np.ndarray,
+        start_index: int,
+        end_index: int,
+        pkl_path: str
+    ) -> pd.core.frame.DataFrame:
+        """
+        Extract all of the running speed data
+
+        Parameters
+        ----------
+        frame_times: numpy.ndarray
+            list of the vsync times
+        start_index: int
+            Index to the first frame of the stimulus
+        end_index: int
+           Index to the last frame of the stimulus
+        pkl_path: string
+            Path to the stimulus pickle file
+
+        Returns
+        -------
+        list[pd.DataFrame, pd.DataFrame]
+            the velocity data and the raw data
+
+        Notes
+        -------
+            velocity pd.DataFrame:
+                columns:
+                    "velocity": computed running speed
+                    "net_rotation": dx in radians
+                    "frame_indexes": frame indexes into
+                        the full vsync times list
+
+            raw data pd.DataFrame:
```

Comment:
Is this referring to an argument that is no longer here? Same with `velocity`

Reply:
I've updated this.